test/cluster/regression/github-15496.td

```diff
@@ -0,0 +1,29 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Regression test for https://github.com/MaterializeInc/materialize/issues/15496
+# This test is special in that it requires MZ_SOFT_ASSERTIONS=0 at present.
+
+# Set data for test up
+> SET cluster = cluster1;
```

Comment:
This file is not self-contained because `cluster1` is created in the enclosing test. It might be better to just run the 6 statements directly from the python file as the other `workflow_test_github_~` do. 

```python
c.testdrive(
    dedent(
        f"""
        # Set data for test up
        > SET cluster = cluster1;
        
        > CREATE TABLE base (data bigint, diff bigint);
        
        // ...
        """
    )
)
```

Reply:
Cool, revised.